styles/components/_c-modal.scss

```diff
@@ -6,7 +6,7 @@
    ========================================================================== */
 
 $au-modal-z-index: 9999 !default;
-$au-modal-backdrop-z-index: 9998 !default;
+$au-modal-backdrop-z-index: 9999 !default;
```

Comment:
Question: Does it still make sense to make this configurable after this change? For this to work properly both z-index values need to be the same I think?

Reply:
indeed it should be the same. I wanted to keep it around in case others would use this property, but if you prefer i can remove it instead?